linera-service/tests/end_to_end_tests.rs

```diff
@@ -1087,19 +1089,18 @@ async fn test_reconfiguration(network: Network) {
 
     // Query balance for first and last user chain
     assert_eq!(client.query_balance(chain_1).await.unwrap(), 10);
-    assert_eq!(client.query_balance(chain_2).await.unwrap(), 10);
+    assert_eq!(client.query_balance(chain_2).await.unwrap(), 0);
 
-    // Transfer 10 units then 5 back
-    client.transfer(10, chain_1, chain_2).await;
-    client.transfer(5, chain_2, chain_1).await;
+    // Transfer 5 units
+    client.transfer(5, chain_1, chain_2).await;
 
     // Restart last server (dropping it kills the process)
     runner.kill_server(4, 3);
     runner.start_server(4, 3).await;
 
     // Query balances again
     assert_eq!(client.query_balance(chain_1).await.unwrap(), 5);
-    assert_eq!(client.query_balance(chain_2).await.unwrap(), 15);
+    assert_eq!(client.query_balance(chain_2).await.unwrap(), 5);
```

Comment:
nit: I would avoid using identical pairs of numbers as it's easier to get them confused. Try something like 7 and 3?